kubetest/local.go

```diff
@@ -52,13 +53,32 @@ func newLocalCluster() *localCluster {
 	}
 }
 
+func (n localCluster) getScript(scriptPath string) (string, error) {
+	cwd, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	path := filepath.Join(cwd, scriptPath)
+	if _, err := os.Stat(path); err == nil {
+		return path, nil
+	}
+	path = filepath.Join(cwd, "kubernetes", scriptPath)
+	if _, err := os.Stat(path); err == nil {
+		return path, nil
+	}
```

Comment:
are those path experimental just to see where to get the script? I'd assume only one of the paths will work.

Reply:
@krzyzacy - yes, we pick the first one that works and use that.

Reply:
you probably can log down which path it picked, and clean up the unneeded one afterwards?

Reply:
@krzyzacy So i am expecting the second one to work in the CI. The first one works in a normal dev box where you checkout k/k and run `make cross`. So we will end up needing both i am afraid. However, let's get this in and confirm the theory.

Reply:
ok :-)